packages/code-du-travail-frontend/src/common/Disclaimer.js

```diff
@@ -1,20 +1,53 @@
-import React from "react";
-import { Alert } from "@cdt/ui";
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import { Toast, theme } from "@cdt/ui";
 
-const Disclaimer = () => (
-  <Alert warning>
-    Important ! Ce site est en cours de construction : les données qui s&apos;y
-    trouvent peuvent être erronées ou imprécises.{" "}
-    <a
-      title="consulter l'ordonance"
-      target="_blank"
-      rel="noopener noreferrer"
-      href="https://www.legifrance.gouv.fr/affichTexteArticle.do;jsessionid=AE9DCF75DDCF0465784CEE0E7D62729F.tplgfr37s_2?idArticle=JORFARTI000035607420&cidTexte=JORFTEXT000035607388&dateTexte=29990101&categorieLien=id"
-    >
-      L&apos;ouverture officielle du site
-    </a>{" "}
-    est prévue pour 2020.
-  </Alert>
-);
+const Disclaimer = () => {
+  const [isDisclaimerHidden, hideDisclaimer] = useState(true);
+  useEffect(() => {
+    hideDisclaimer(sessionStorage.getItem("isDisclaimerHidden"));
+  });
+  if (isDisclaimerHidden) return null;
+  return (
+    <Wrapper>
+      <Toast
+        type="warning"
+        wide
+        shadow
+        animate="from-bottom"
+        onRemove={() => {
+          sessionStorage.setItem("isDisclaimerHidden", true);
+          hideDisclaimer(true);
```

Comment:
I would prefer setDisclaimerHidden() 
hideDisclaimer(false) does'nt hide the disclainer but show it which could be counter-intuitive ;)

Reply:
👍